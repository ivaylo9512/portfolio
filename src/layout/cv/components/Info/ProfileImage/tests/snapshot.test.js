import { shallow } from 'enzyme';
import PorfileImage from '../ProfileImage';

describe('PorfileImage snapshot tests', () => {
    const createWrapper = () => shallow(
        <PorfileImage />
    );

    it('should match snapshot', () => {
        const wrapper = createWrapper();

        expect(wrapper).toMatchSnapshot();
    })
})