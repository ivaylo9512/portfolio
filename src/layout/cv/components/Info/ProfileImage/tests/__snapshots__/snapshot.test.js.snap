// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PorfileImage snapshot tests should match snapshot 1`] = `
<div
  className="image-container"
>
  <img
    alt="profile"
    src="profile-image.png"
  />
</div>
`;
